src/colorizer/CanvasOverlay.ts

```diff
@@ -1,38 +1,96 @@
+import { Vector2 } from "three";
 import { numberToSciNotation } from "./utils/math_utils";
 
-export type ScaleBarOptions = {
-  minWidthPx: number;
+type StyleOptions = {
   fontSizePx: number;
   fontFamily: string;
   fontColor: string;
+  fontStyle: string;
+};
+
+export type ScaleBarOptions = StyleOptions & {
+  minWidthPx: number;
   visible: boolean;
   unitsPerScreenPixel: number;
   units: string;
 };
 
-const defaultScaleBarOptions: ScaleBarOptions = {
-  minWidthPx: 80,
+export type TimestampOptions = StyleOptions & {
+  visible: boolean;
+  maxTimeSec: number;
+  currTimeSec: number;
+  frameDurationSec: number;
+  startTimeSec: number;
+};
+
+export type BackgroundOptions = {
```

Comment:
do these other options types need to be exported?  ScaleBarOptions, TimestampOptions, BackgroundOptions?

Reply:
I guess not, because they only ever define parameter types accepted by the CanvasOverlay. Updated!